src/map/clif.cpp

```diff
@@ -1926,6 +1926,17 @@ void clif_homskillinfoblock( homun_data& hd ){
 	}
 
 	clif_send( packet, packet->packetLength, &sd->bl, SELF );
+
+	if (battle_config.display_status_timers) {
```

Comment:
try change this to upper loop, and clif 0 tick only if skill is not in the vector
```
	for(int i = 0, count = 0; i < MAX_HOMUNSKILL; i++) {
		(...)
			if (battle_config.display_status_timers) {
				if (!util::vector_exists(hd.blockskill, id))
					clif_skill_cooldown(*sd, id, 0);
			}
		}
```

Reply:
That wouldn't work, sending ZC_HOSKILLINFO_LIST makes the client refresh the timers. In fact, sending it twice somehow makes the timers revert to what the client has internally.